src/Goerli-DssSpell.t.sol

```diff
@@ -194,30 +246,30 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        checkChainlogKey("STARKNET_GOV_RELAY_LEGACY");
-        checkChainlogKey("STARKNET_GOV_RELAY");
-        checkChainlogKey("MCD_CLIP_CALC_GUSD_A");
-        checkChainlogKey("MCD_CLIP_CALC_USDC_A");
-        checkChainlogKey("MCD_CLIP_CALC_PAXUSD_A");
+        checkChainlogKey("GNO");
+        checkChainlogKey("PIP_GNO");
+        checkChainlogKey("MCD_JOIN_GNO_A");
+        checkChainlogKey("MCD_CLIP_GNO_A");
+        checkChainlogKey("MCD_CLIP_CALC_GNO_A");
```

Comment:
Need to check for RWA additions.

Reply:
Added at https://github.com/makerdao/spells-goerli/commit/8a4466dd96044e65ef7efa05992b26d67b5789b2